core/vm/eips.go

```diff
@@ -307,26 +307,26 @@ func enable6780(jt *JumpTable) {
 
 func enable4762(jt *JumpTable) {
 	jt[SSTORE].constantGas = 0
-	jt[SSTORE].dynamicGas = gasSStore4762
+	jt[SSTORE].dynamicGas = nil
 	jt[SLOAD].constantGas = 0
-	jt[SLOAD].dynamicGas = gasSLoad4762
-	jt[BALANCE].dynamicGas = gasBalance4762
+	jt[SLOAD].dynamicGas = nil
+	jt[BALANCE].dynamicGas = nil
 	jt[BALANCE].constantGas = 0
 	jt[EXTCODESIZE].constantGas = 0
-	jt[EXTCODESIZE].dynamicGas = gasExtCodeSize4762
+	jt[EXTCODESIZE].dynamicGas = nil
 	jt[EXTCODEHASH].constantGas = 0
-	jt[EXTCODEHASH].dynamicGas = gasExtCodeHash4762
+	jt[EXTCODEHASH].dynamicGas = nil
 	jt[EXTCODECOPY].constantGas = 0
-	jt[EXTCODECOPY].dynamicGas = gasExtCodeCopyEIP4762
-	jt[SELFDESTRUCT].dynamicGas = gasSelfdestructEIP4762
+	jt[EXTCODECOPY].dynamicGas = gasExtCodeCopy
+	jt[SELFDESTRUCT].dynamicGas = nil
 	jt[CREATE].constantGas = params.CreateNGasEip4762
 	jt[CREATE2].constantGas = params.CreateNGasEip4762
 	jt[CALL].constantGas = 0
-	jt[CALL].dynamicGas = gasCallEIP4762
+	jt[CALL].dynamicGas = gasZero
```

Comment:
you should not need to do this, `nil` works just as well

Reply:
Yes, I know. But if I used `nil` for `*CALL` the testing library crashed. That didn't make any sense to me but it's true. I didn't want to debug that since eventually we'll use a PR that wont' have `nil` for `*CALL`s so this is temporary.